disk/tests/test_check.py

```diff
@@ -20,7 +20,11 @@
     'system.disk.free',
     'system.disk.in_use',
     'system.disk.total',
-    'system.disk.used'
+    'system.disk.used',
+]
+DISK_RATES = [
+    'system.disk.write_time_pct',
```

Comment:
I'm a bit confused these two appear as `gauges` in the metadata?

Reply:
Honestly I didn't pay much attention, I only assumed this https://github.com/DataDog/integrations-core/pull/1018 introduced something we needed to test but you're right